### DISCUSSION TITLE
Pure docker continer

### DISCUSSION THREAD

**Discussion by starrider346** [pelican-dev/panel#113] [Ideas]
Aswell as eggs. Sometimes a server can be simply a docker image. Is it possible to have choice to choose to use an egg or just a docker image?

**Comment by TekExplorer** [comment]
As far as i understand, pterodactyl and pelican eggs require modified images to work, so it may not be so simple.

**Reply by starrider346** [reply]
I don't mean eggs. I mean as well as eggs, you can choose to run just a docker image. Like portainer.

**Reply by TekExplorer** [reply]
thats what i'm saying. pelican requires modified images, so normal images wont work.
just... use portainer for that.

**Comment by parkervcp** [comment]
The way we handle the entrypoint, and commands this is not really viable.

**Comment by lancepioch** [comment]
Sorry, I don't believe this is actually possible!
